src/test/java/com/purbon/kafka/topology/integration/containerutils/ConnectContainer.java

```diff
@@ -8,7 +8,7 @@
 public class ConnectContainer extends GenericContainer<ConnectContainer> {
 
   private static final DockerImageName DEFAULT_IMAGE =
-      DockerImageName.parse("confluentinc/cp-kafka-connect").withTag("6.0.2");
+      DockerImageName.parse("confluentinc/cp-kafka-connect").withTag("7.5.0");
```

Comment:
Probably should come from system property to be able to compare various CP versions. Not that important now.

Reply:
I agree, but I think that would be a separate PR. There are several, similar hard-coded versions around.